src/app/Http/Requests/PostRequest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PostRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
```

Comment:
do you think this authorize function is doing anything in this context?

Reply:
As of now po sir, wala pa po.

Reply:
if none, remove it thanks

Reply:
noted on that po, sir. thank you po.

Reply:
remove?

Reply:
I have resolved this comment, sir. Thank you! 🙇